android-core/src/main/java/com/mparticle/MParticleOptions.java

```diff
@@ -179,6 +181,16 @@ public MParticle.Environment getEnvironment() {
         return mEnvironment;
     }
 
+    /**
+     * Get list of local kits
```

Comment:
All uses of the term "local kits" in comments and variable names should be "sideloaded kits" to be in line with the other platforms